src/dialects/abstract/query-interface.d.ts

```diff
@@ -214,6 +214,15 @@ export interface IndexesOptions {
 
 export interface QueryInterfaceIndexOptions extends IndexesOptions, QueryInterfaceOptions {}
 
+/**
+ * Interface for bulk delete options
+ */
+export interface QueryInterfaceBulkDeleteOptions {
```

Comment:
These options are passed to `Sequelize#query` so should inherit from `QueryOptions`

Reply:
That's a good one